public_dropin_gpu_environments/python311_genai/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# This is the default base image for use with user models and workflows.
+# It contains a variety of common useful data-science packages and tools.
+FROM datarobot/dropin-env-base-jdk:ubi8.8-py3.11-jdk11.0.22-drum1.11.1-mlops9.2.8
+
+# Install the list of core requirements, e.g. sklearn, numpy, pandas, flask.
+# **Don't modify this file!**
+COPY dr_requirements.txt dr_requirements.txt
+
+# '--upgrade-strategy eager' will upgrade installed dependencies
+# according to package requirements or to the latest
+RUN pip3 install -U pip && \
+    pip3 install -U --upgrade-strategy eager --no-cache-dir -r dr_requirements.txt  && \
+    rm -rf dr_requirements.txt
+
+# Install the list of custom Python requirements, e.g. keras, xgboost, etc.
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cu118 && \
```

Comment:
Changed from `https://download.pytorch.org/whl/cpu` in regular `python311_genai`  to https://download.pytorch.org/whl/cu118 in GPU enabled one